node/src/exchange/utils.rs

```diff
@@ -56,6 +67,20 @@ impl HeaderResponseExt for HeaderResponse {
             }
         }
     }
+
+    // TODO: how forthcoming should we be with errors and description?
+    fn not_found() -> HeaderResponse {
+        HeaderResponse {
+            body: NOT_FOUND_MSG.as_bytes().to_vec(),
```

Comment:
Is Celestia encoding an error message in body?

Reply:
[They do not](https://github.com/celestiaorg/go-header/blob/a6544e3c7354c4b798fe96ed47d8a537bac03ca3/p2p/server.go#L134-L136) encode error messages. So `body` should be empty `Vec`.

Reply:
changed 👍